userguide/object_detection/data-preparation.md

```diff
@@ -0,0 +1,133 @@
+# IG02 Data Preparation
+
+In this section, we will show you how to download a publicly available dataset
+and load it into an SFrame. This will allow you to try out the Object Detection
+toolkit for yourself. To get this dataset into the format expected by our
+toolkit, we will rely on many useful SFrame functions.
+
+*Note: Please make sure that you have Turi Create 4.3 or above for these steps*
+
+The dataset that we will use is [INRIA Annotations for Graz-02
+(IG02)](https://lear.inrialpes.fr/people/marszalek/data/ig02/). Our goal is to
```

Comment:
We just need a reference to the dataset disclaimer, after the period on this line. Thanks!
`[<sup>1</sup>](../datasets.md)`

Reply:
Good idea. I changed it to use markdown footnotes, which looks good.

Reply:
At least through gitbook - GitHub viewing is not as nice, but functionally OK.